thirdparty/jaspr_bootstrap/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The other packages use a symlink to the topmost license, if thats ok for you I would do this here too.

Reply:
I updated the license.